### ISSUE TITLE
Flaky test?

### ISSUE THREAD

**Issue by pavolloffay** [open-telemetry/opentelemetry-operator#2214]
### Component(s)

opamp bridge

### What happened?

https://github.com/open-telemetry/opentelemetry-operator/actions/runs/6484221330/job/17607557637#step:4:1662

### Kubernetes Version

all

### Operator version

main

### Collector version

main

## Environment
OS: (e.g., "Ubuntu 20.04")
Compiler(if manually compiled): (e.g., "go 14.2")

### Log output

```shell
=== RUN   TestAgent_onMessage/base_case_http
    agent_test.go:503: 
        	Error Trace:	/home/runner/work/opentelemetry-operator/opentelemetry-operator/cmd/operator-opamp-bridge/agent/agent_test.go:503
        	Error:      	Not equal: 
        	            	expected: &protobufs.RemoteConfigStatus{state:impl.MessageState{NoUnkeyedLiterals:pragma.NoUnkeyedLiterals{}, DoNotCompare:pragma.DoNotCompare{}, DoNotCopy:pragma.DoNotCopy{}, atomicMessageInfo:(*impl.MessageInfo)(nil)}, sizeCache:0, unknownFields:[]uint8(nil), LastRemoteConfigHash:[]uint8{0x67, 0x6f, 0x6f, 0x64, 0x2f, 0x74, 0x65, 0x73, 0x74, 0x6e, 0x61, 0x6d, 0x65, 0x73, 0x70, 0x61, 0x63, 0x65, 0x34, 0x30, 0x35}, Status:1, ErrorMessage:""}
        	            	actual  : &protobufs.RemoteConfigStatus{state:impl.MessageState{NoUnkeyedLiterals:pragma.NoUnkeyedLiterals{}, DoNotCompare:pragma.DoNotCompare{}, DoNotCopy:pragma.DoNotCopy{}, atomicMessageInfo:(*impl.MessageInfo)(nil)}, sizeCache:0, unknownFields:[]uint8(nil), LastRemoteConfigHash:[]uint8{0x67, 0x6f, 0x6f, 0x64, 0x2f, 0x74, 0x65, 0x73, 0x74, 0x6e, 0x61, 0x6d, 0x65, 0x73, 0x70, 0x61, 0x63, 0x65, 0x34, 0x30, 0x31}, Status:1, ErrorMessage:""}
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -14,3 +14,3 @@
        	            	   00000000  67 6f 6f 64 2f 74 65 73  74 6e 61 6d 65 73 70 61  |good/testnamespa|
        	            	-  00000010  63 65 34 30 35                                    |ce405|
        	            	+  00000010  63 65 34 30 31                                    |ce401|
        	            	  },
        	Test:       	TestAgent_onMessage/base_case_http
```
```


### Additional context

_No response_

**Comment by pavolloffay** [comment]
cc) @jaronoff97

**Comment by jaronoff97** [comment]
Yep I have a fix coming

**Comment by jaronoff97** [comment]
closed in #2217
